libs/core/src/lib/shellbar/user-actions-menu/user-actions-menu-item/user-actions-menu-item.component.scss

```
@@ -0,0 +1,19 @@
+.fd-user-actions-menu-item {
+    &__custom-icon.fd-list__icon[class*=sap-icon] {
```

I think `^` will be better and there will be less chance for name collision.
```suggestion
    &__custom-icon.fd-list__icon[class^=sap-icon] {
```